ansible/inventory.yml

```diff
@@ -151,8 +155,28 @@ hosts:
         ubuntu1804-x64-1: {ip: 165.225.149.88, user: ubuntu}
 
     - macstadium:
-        macos11.0-arm64-1: {ip: 199.7.163.9, user: administrator}
-        macos11.0-arm64-2: {ip: 199.7.163.10, user: administrator}
+        macos11.0-arm64-1:
+            ip: 199.7.163.9
+            user: administrator
+            remote_env:
+                PATH: /opt/homebrew/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin
+        macos11.0-arm64-2:
+            ip: 199.7.163.10
+            user: administrator
+            remote_env:
+                PATH: /opt/homebrew/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin
```

Comment:
Are these the DTK's, which are going/gone?

Reply:
yes forgot about that - just rebased the PR whilst setting up parts of the release machine for me to test - https://github.com/nodejs/node/pull/37861

will remove them tomorrow and land the PR then